delta/ml/ml_config.py

```diff
@@ -272,6 +272,27 @@ def dir(self) -> str:
             tbd = os.path.join(appdirs.AppDirs('delta', 'nasa').user_data_dir, 'tensorboard')
         return tbd
 
+class ClassifyConfig(config.DeltaConfigComponent):
+    """
+    Configure classification options.
+    """
+    def __init__(self):
+        super().__init__()
+        self.register_field('regions', list, None, None,
```

Comment:
I added sample files in the new /docs folder.  Currently a lot of our documentation is in the /delta/config/README file instead of the base level README file so I wasn't sure where to reference the sample files from.